src/macros.rs

```diff
@@ -0,0 +1,288 @@
+/// Construct a `plist::Value` from a JSON literal.
```

Comment:
How about "Construct a `plist::Value` from a JSON-like literal."? As some things will serialise differently to JSON.